base_export_async/models/delay_export.py

```diff
@@ -101,40 +103,30 @@ def export(self, params):
         )
         date_today = fields.Date.today()
         expiration_date = fields.Date.to_string(
-            date_today + relativedelta(days=+int(time_to_live))
+            date_today + relativedelta(days=int(time_to_live) + 1)
```

Comment:
i think we should keep date_today + relativedelta(days=int(time_to_live))

Reply:
We should plus one day for the expiration date, otherwise, the record might be expired unexpectedly.

For example, I set time_to_live as 7 days, I expected the record to be available until the end of the 7th day.

Reply:
For this kind of change, could you please add a test that fails without it?
It would be easier to understand what you are trying to fix/improve.

Reply:
I think this improvement should not be placed in this migration PR. I reverted the related changes.